Polish/przygoda/jedzenie/biznes/biznes.md

```diff
@@ -1 +1,5 @@
 Chcesz wykorzystać potencjał ptasiego mleczka i jeszcze na tym zarobić. Zabierasz się za biznesplan! Co robisz?
+
+[Zaczynasz pisać i pisać...](cukrzyca/cukrzyca.md)
+
+[Wstrzymujesz pisanie i szukasz klientow na mleczko](podlakorp/podklakorp.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.